test/views/benefits/income/IncomeTaxBenefitsAmountViewSpec.scala

```diff
@@ -36,7 +36,6 @@ class IncomeTaxBenefitsAmountViewSpec extends ViewUnitTest {
   private val expectedErrorHref = "#amount"
 
   object Selectors {
-    def paragraphSelector: String = "#main-content > div > div > p"
 
     def paragraphSelector2(index: Int): String = s"#main-content > div > div > p:nth-child($index)"
```

Comment:
Not sure if it will break too many things, hopefully not, but maybe if we got rid of the `paragraphSelector` method, we can refactor the `paragraphSelector2` to just `paragraphSelector`? I am assuming that `2` is because there is a second method, and not anything else.

Reply:
paragraphSelector has been removed and paragraphSelector2 renamed